tests/tasks.py

```diff
@@ -3,18 +3,27 @@
 import os
 import os.path
 from time import sleep
-
 from invoke import task
-
-
-GLIDE_ARCH = 'linux-amd64'
+from invoke import exceptions
 
 
 @task
-def glide(c, arch=GLIDE_ARCH, version='v0.13.2'):
-    """Dowload glide."""
+def glide(c, version='v0.13.2'):
+    """Download glide."""
     print('Downloading glide')
 
+    # find glide version, copied from glide installation script
+    os_name_res = c.run('uname|tr "[:upper:]" "[:lower:]"', hide='stdout')
+    os_name = os_name_res.stdout.replace('\n', '')
+    if os_name == 'linux':
+        arch = 'linux-amd64'
+    elif os_name == 'darwin':
+        arch = 'darwin-amd64'
```

Comment:
try this instead https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on